src/components/buttons/secondary-button/secondary-button.js

```diff
@@ -46,10 +46,17 @@ export const SecondaryButton = props => {
     `,
     getStateStyles(props.isDisabled, isActive, props.theme),
     getThemeStyles(props.theme),
+    shouldUseLinkTag &&
+      css`
+        text-decoration: none;
+      `,
   ];
 
-  const containerElements = (
+  const linkProps = shouldUseLinkTag ? { to: props.linkTo, as: Link } : {};
```

Comment:
Nice migration strategy. Maybe a deprecation warning ontop would make it silky smooth.

Reply:
Genau